tests/test_Summary.cpp

```diff
@@ -1477,13 +1484,13 @@ BOOST_AUTO_TEST_CASE(BLOCK_VARIABLES) {
     BOOST_CHECK_CLOSE( 31.0  , units.to_si( UnitSystem::measure::viscosity , ecl_sum_get_general_var( resp, 1, "BVOIL:1,1,1")) , 1e-5);
     BOOST_CHECK_CLOSE( 33.0  , units.to_si( UnitSystem::measure::viscosity , ecl_sum_get_general_var( resp, 1, "BOVIS:1,1,1")) , 1e-5);
 
-    BOOST_CHECK_CLOSE( 100                , ecl_sum_get_well_completion_var( resp, 1, "W_1", "CTFAC", 1, 1, 1), 1e-5);
-    BOOST_CHECK_CLOSE( 2.1430730819702148 , ecl_sum_get_well_completion_var( resp, 1, "W_2", "CTFAC", 2, 1, 1), 1e-5);
-    BOOST_CHECK_CLOSE( 2.6788413524627686 , ecl_sum_get_well_completion_var( resp, 1, "W_2", "CTFAC", 2, 1, 2), 1e-5);
```

Comment:
Why has the numerical value we are comparing with changed?

Reply:
Because we now get the values from the synthetic [`data::Connection`](https://github.com/OPM/opm-common/blob/2232bb08199fd3319b8c0b989dcdb97837b6ac33/tests/test_Summary.cpp#L256-L260) instead the `Connection` objects produced by parsing the input file.

In actual runs the values will be obtained from [the `Schedule`](https://github.com/OPM/opm-simulators/blob/d3c86e2303c6f691e808036ccdbfc7cae123a371/opm/simulators/wells/WellState.hpp#L208), though possibly adjusted by `WELPI` or similar dynamic effects.

Reply:
Good - I understand!